src/components/listbox/listbox.jsx

```diff
@@ -17,18 +18,44 @@ export default function Listbox(props) {
     }
   });
 
+  const refIndexOf = target => props.refs.findIndex(ref => ref.current && ref.current.contains(target));
+  const optionsWasSelected = (toggledRefIndex, isSelected) => {
+    let newSelectedIndexes = selectedIndexes.slice(0, selectedIndexes.length);
+
+    if (isSelected) {
+      newSelectedIndexes.push(toggledRefIndex);
+    } else {
+      const indexOfToggledRef = newSelectedIndexes.indexOf(toggledRefIndex);
+      newSelectedIndexes = newSelectedIndexes.splice(indexOfToggledRef, 1);
+    }
+
+    if (props.selectionLimit && newSelectedIndexes.length > props.selectionLimit) {
```

Comment:
I think we might want to remove `selectionLimit` because we don't have a design for it. I don't think the existing Mavenlink application has any dropdown with a limited set of selected options. It is either: 1 or `multiple`